lib/msal-node/test/client/ManagedIdentitySources/AppService.spec.ts

```diff
@@ -72,6 +77,15 @@ describe("Acquires a token successfully via an App Service Managed Identity", ()
             expect(networkManagedIdentityResult.accessToken).toEqual(
                 DEFAULT_USER_SYSTEM_ASSIGNED_MANAGED_IDENTITY_AUTHENTICATION_RESULT.accessToken
             );
+
+            const url: URLSearchParams = new URLSearchParams(
+                sendGetRequestAsyncSpy.mock.lastCall[0]
+            );
+            expect(
+                url.has(
+                    ManagedIdentityUserAssignedIdQueryParameterNames.MANAGED_IDENTITY_CLIENT_ID_2019
```

Comment:
even in tests you can use MANAGED_IDENTITY_CLIENT_ID

Reply:
done